soap/src/java/com/zimbra/soap/mail/type/ModifySearchFolderSpec.java

```diff
@@ -24,6 +24,9 @@
 
 import com.zimbra.common.soap.MailConstants;
 
+import io.leangen.graphql.annotations.GraphQLInputField;
+import io.leangen.graphql.annotations.GraphQLNonNull;
+
 @XmlAccessorType(XmlAccessType.NONE)
 public class ModifySearchFolderSpec {
```

Comment:
shouldn't there by any graph QL type annotation here?

Reply:
this class only has input fields, so type annotation is not needed here

Reply:
You should still annotate the class with a type annotation and description that explains this is used as "Input for modifying a search folder." or something.

Reply:
added